stdlib/public/core/StringStorageBridge.swift

```diff
@@ -335,6 +373,464 @@ extension __SharedStringStorage {
     // reference will make the instance non-unique.
     return self
   }
+  
+  @objc(copy)
+  final internal func copy() -> AnyObject {
+    return self
+  }
+  
+  @objc(mutableCopyWithZone:)
+  final internal func mutableCopy(with zone: _SwiftNSZone?) -> AnyObject {
+    return _SwiftNSMutableString(self.asString)
+  }
+  
+  @objc(mutableCopy)
+  final internal func mutableCopy() -> AnyObject {
+    return _SwiftNSMutableString(self.asString)
+  }
+  
+}
+
+@usableFromInline
```

Comment:
Eek, why is this ABI?

Reply:
Hmm. I have this uncomfortable memory of this being needed for some sort of "make sure the compiler emits it" reason, but I can't bring it to mind. I can try removing it.